reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java

```diff
@@ -107,6 +107,52 @@ public void set_Status(int _Status) {
         this._Status = _Status;
     }
     
+    private int _BitStatus=0;
```

Comment:
if the old status and related methods are not used any more, they should be removed and BitStatus could be renamed to Status